web/php/Screens/Feed/FrontForumFeed.php

```diff
@@ -126,38 +126,38 @@ public function build($runData)
         $categories = array();
 
         if (count($cats)>20) {
-            throw new ProcessException(_("Maximum number of categories exceeded."), "max_categories");
+            throw new ProcessException(_('Maximum number of categories exceeded.'), 'max_categories');
```

Comment:
I just tested this, and Wikidot no longer as a maximum category limit (or that limit is well above 20). I created an issue to get rid of this: (TODO: internet problems)